src/reaper_osara.cpp

```diff
@@ -3583,6 +3583,26 @@ void cmdUnselAllTracksItemsPoints(Command* command) {
 		outputMessage(translate("unselected tracks/items/envelope points"));
 }
 
+void reportTempoTimeSig() {
+ReaProject* proj=EnumProjects(-1, nullptr, 0);
+if(proj==nullptr) return;
+double tempo=0;
+int timesig1=0;
```

Comment:
Please rename these to be more descriptive; e.g. num and denom. This makes it clearer what each of these actually is: numerator and denominator.